cairo-sys-rs/build.rs

```diff
@@ -70,3 +74,43 @@ fn find(package_name: &str, shared_libs: &[&str]) -> Result<(), Error> {
         Err(err) => Err(err),
     }
 }
+
+// HACK: if libffi is installed via homebrew it doesn't get linked correctly.
+// this adds the homebrew libffi to the pkg-config-path.
+#[cfg(target_os = "macos")]
+fn macos_maybe_find_lib_ffi() {
+    use std::path::{Path, PathBuf};
+    use std::process::Command;
+    const PKG_CONFIG_PATH: &str = "PKG_CONFIG_PATH";
```

Comment:
Small nit: Please add an empty line after the imports, it makes code easier to read.